artifacthub-pkg.yml

```
@@ -0,0 +1,27 @@
+---
+version: 0.1.0
+name: deprecated-api-versions
+displayName: Deprecated API Versions
+createdAt: '2022-07-19T16:39:10+02:00'
```

```suggestion
createdAt: '2022-10-26T16:39:10+02:00'
```

Signed-off-by: Víctor Cuadrado Juan <vcuadradojuan@suse.de>